crates/tests-integration/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tests-integration"
-version = "0.0.0"
+version = "0.1.0-alpha.8-pre"
```

Comment:
Is this a mistake?

Reply:
@th4s Thanks for catching this. I found one more in the examples crate